setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="razorpay",
-    version="0.1.3",
+    version="0.2.0",
```

Comment:
version bumps shouldn't happen in a feature PR, ideally (think multiple features being worked on). You only bump the version on master once you are ready to release.